nservicebus/upgrades/5to6.md

```diff
@@ -39,19 +39,21 @@ The endpoint name is used as a logical identifier when sending or receiving mess
 
 ## Message handlers
 
-The handler method on `IHandleMessages<T>` now returns a Task. In order to leverage async code, add the `async` keyword to the handler method and use `await` for async methods. In order to convert the synchronous code add `return Task.FromResult(0);` to the end of the handler methods.
+The handler method on `IHandleMessages<T>` now returns a Task. In order to leverage async code, add the `async` keyword to the handler method and use `await` for async methods. In order to convert the synchronous code add `return Task.FromResult(0);` or `return Task.CompletedTask` (.NET 4.6 and higher) to the end of the handler methods.
```

Comment:
> to the end of the handler methods

i know u didnt change this but "to the end of the handler methods" doesnt seem to make sense